src/calculator/Calculator.java

```diff
@@ -1,35 +1,37 @@
+/*
+    @Class Calculator contains the methods for arithmetic calculations
+
+    @calculator() uses the required switch statement used to identify
+    the operation and calls the arithmetic methods
+
+    @Arithmetic methods add -- ads two numbers, subtract -- deducts two numbers,
+    multiply -- multiplies two numbers, divide -- divides two number
+
+    @all arithmetic methods return double type values and use two double type
+
+ */
 package calculator;
 
-import java.util.Scanner;
 import java.lang.*;
 
-public  class Calculator {
 
-    public  double calculator() {
-        double firstNumber;
-        double lastNumber;
-        String operation = new String("");
+public  class Calculator {
 
-        firstNumber = inputVariables("first");
-        lastNumber = inputVariables("second");
+    public  double calculator(double n1, double n2, String op) {
```

Comment:
not ok, partea asta e input, nu are treaba cu calculatorul, in calculator ai doar operatiile matematice